client/state/ui/editor/post/actions.js

```diff
@@ -12,182 +13,190 @@ export function setEditingMode( mode, modeTitle, site ) {
 
 export function startEditingNew( site, postOptions ) {
 	debug( 'startEditingNew', site, postOptions );
-	return { type: 'TODO' };
+	return {
+		type: EDITOR_POST_CURRENT_ID,
+		siteId: site.ID,
+		postId: null
+	};
 }
 
 export function startEditingExisting( site, postID ) {
 	debug( 'startEditingExisting', site, postID );
-	return { type: 'TODO' };
+	return {
+		type: EDITOR_POST_CURRENT_ID,
+		siteId: site.ID,
+		postId: postID
+	};
 }
 
-export function toggleStickyStatus( currentStatus ) {
-	debug( 'toggleStickyStatus', currentStatus );
+export function toggleStickyStatus( siteId, postId, currentStatus ) {
```

Comment:
The current plan with #3183 was to have `siteId` and `postId` be the last arguments, probably so that it could be called as either:

```jsx
editPost( { title: 'Hello World' }, 2916284, 112 );

// or, without post ID:

editPost( { title: 'Hello World' }, 2916284 )
```

If we were to have a "currently edited post ID" selector, I guess this is less important.

In the end, we should just be sure to be on the same page. Any opinions from you one way or the other?